cookbooks/aws-parallelcluster-environment/libraries/directory_service_utils.rb

```diff
@@ -0,0 +1,24 @@
+#
+# Parse parameter DomainReadOnlyUser: cn=ReadOnlyUser,ou=Users,ou=CORP,dc=corp,dc=sirena,dc=com
+# and return the actual Name of the ReadOnly user
+def _get_cn_subparam(param)
```

Comment:
you can unit test it, see: cookbooks/aws-parallelcluster-environment/spec/unit/libraries/volume_spec.rb